db/bigtable_eth1.go

```diff
@@ -1154,6 +1154,54 @@ func decodeIsContractUpdateTs(ts gcp_bigtable.Timestamp) (block_number, tx_idx,
 	return block_number, tx_idx, trace_idx
 }
 
+func (bigtable *Bigtable) TransformContract(blk *types.Eth1Block, cache *freecache.Cache) (bulkData *types.BulkMutations, bulkMetadataUpdates *types.BulkMutations, err error) {
+	bulkData = &types.BulkMutations{}
+	bulkMetadataUpdates = &types.BulkMutations{}
+	ContractUpdateWrites := &types.BulkMutations{}
```

Comment:
**nitpick:** `ContractUpdateWrites` should have a lowercase first letter, `contractUpdateWrites`.